### ISSUE TITLE
App Load Order?

### ISSUE THREAD

**Issue by robmoggach** [torchbox/wagtail-markdown#6]
Not sure where this is coming from... possibly django 1.9

```
File "/Users/rob/Sites/moggach/apps/wagtailmarkdown/__init__.py", line 16, in <module>
  from wagtail.wagtailcore.blocks import TextBlock
File "/Users/rob/.virtualenvs/wagtail/lib/python2.7/site-packages/wagtail/wagtailcore/blocks/__init__.py", line 5, in <module>
  from .field_block import *  # NOQA
File "/Users/rob/.virtualenvs/wagtail/lib/python2.7/site-packages/wagtail/wagtailcore/blocks/field_block.py", line 15, in <module>
  from wagtail.wagtailcore.rich_text import RichText
File "/Users/rob/.virtualenvs/wagtail/lib/python2.7/site-packages/wagtail/wagtailcore/rich_text.py", line 10, in <module>
  from wagtail.wagtailcore.models import Page
File "/Users/rob/.virtualenvs/wagtail/lib/python2.7/site-packages/wagtail/wagtailcore/models.py", line 9, in <module>
  from modelcluster.models import ClusterableModel, get_all_child_relations
File "/Users/rob/.virtualenvs/wagtail/lib/python2.7/site-packages/modelcluster/models.py", line 152, in <module>
  class ClusterableModel(models.Model):
File "/Users/rob/.virtualenvs/wagtail/lib/python2.7/site-packages/django/db/models/base.py", line 94, in __new__
  app_config = apps.get_containing_app_config(module)
File "/Users/rob/.virtualenvs/wagtail/lib/python2.7/site-packages/django/apps/registry.py", line 239, in get_containing_app_config
  self.check_apps_ready()
File "/Users/rob/.virtualenvs/wagtail/lib/python2.7/site-packages/django/apps/registry.py", line 124, in check_apps_ready
  raise AppRegistryNotReady("Apps aren't loaded yet.")
django.core.exceptions.AppRegistryNotReady: Apps aren't loaded yet.
```

**Comment by ftarnell** [comment]
Hello,

Could you please try the `mediafix` branch and see if it fixes the problem?  (I don't have a working application here at the moment with wagtail-markdown + Django 1.9.)

I've moved the classes out of `__init__.py` into `fields.py`, so you'll need to change imports.  I also reworked JS/CSS handling so it no longer uses/requires `wagtail_hooks.py` and doesn't require the `markdown` class to be set for the editor to work anymore.

Thanks.

**Comment by robmoggach** [comment]
I also submitted a pull request -

--
Robert Moggach
Sr. VFX Supervisor / Flame Artist
http://moggach.com
https://www.linkedin.com/profile/view?id=1664843
Mobile (310) 773-1793



On Wed, Jan 20, 2016 at 10:40 AM, Felicity Tarnell <notifications@github.com

**Comment by robmoggach** [comment]
ok - mediafix seems good but `fontawesome` should be listed as a dependency somewhere. I'll submit a bug. I'd say merge `mediafix` branch.

**Comment by tm-kn** [comment]
Fixed in #19
